### PR TITLE
Removal of authorship tags in source files.

### LINKED ISSUE #104: Remove obsolete comments (authorship)
The code is blighted with nasty old obsolete comments and stuff, such as `__authors__`. We should remove it and never introduce it again.

### PR DESCRIPTION
**Pull request by JiriPavela**
This PR removes authorship and co-authorship "tags" in source files as they are out of date and tedious to maintain. Let's rely on git blame from now on.

Fixes #104.

### CODE REVIEW

File: perun/templates/postprocess.__init__.jinja2 [hunk 1]
```diff
@@ -4,7 +4,3 @@ TODO: Write long docstring of the new postprocessor
 """
 
 SUPPORTED_PROFILES = ["{{ supported_types|join('\", \"') }}"]
-
-{% if author is defined %}
```
Comment (tfiedor) [inline perun/templates/postprocess.__init__.jinja2:8]: This is actually bound to a certain parameter in CLI in (I believe) utils_group? Can you remove the parameter as well in this PR? Maybe you did, but there is so much changes, that I just skimmed it.

**Review by tfiedor — APPROVED** [review]
That was fast. There is just one small issues I believe: removing __author__ from template is bound to some CLI parameter, which is hence useless and should be removed as well. Thanks.

File: perun/templates/postprocess.__init__.jinja2 (hunk 1 above)
Comment (JiriPavela) [reply perun/templates/postprocess.__init__.jinja2:8]: Oh, good catch. I think I found it.

Comment (JiriPavela) [reply perun/templates/postprocess.__init__.jinja2:8]: The new commit should contain a fix.

Comment (tfiedor) [reply perun/templates/postprocess.__init__.jinja2:8]: Good, feel free to merge.

**Pull request merged**